easybuild/tools/config.py

```diff
@@ -493,6 +493,8 @@ class ConfigurationVariables(BaseConfigurationVariables):
         'buildpath',
         'config',
         'containerpath',
+        'failed_installs_build_dirs_path',
```

Comment:
Quite long name. Maybe drop the double plural which sounds a bit odd? E.g. `failed_install_build_dirs_path`
```

Reply:
It's long, but I like that they're grouped together (in output of `--help`, etc.)

I've renamed the options to avoid double plural though, see f63a74379